### PR TITLE
Gamepiece detector

### PR DESCRIPTION
**Pull request by Coolearthsky**

### CODE REVIEW

File: comp/vision/gamepiece_finder24.py [hunk 1]
```diff
@@ -50,7 +60,7 @@ def initialize_nt(self):
 
         # work around https://github.com/robotpy/mostrobotpy/issues/60
         self.inst.getStructTopic("bugfix", NotePosition).publish().set(
-            NotePosition(Translation2d())
+            NotePosition(int(),int())
```
Comment (truher) [inline comp/vision/gamepiece_finder24.py:63]: instead of "int()" here i think you just mean "0"

File: comp/vision/gamepiece_finder24.py [hunk 2]
```diff
@@ -73,13 +83,17 @@ def find_object(self, img):
         for c in contours:
             _, _, cnt_width, cnt_height = cv2.boundingRect(c)
             if (cnt_height < 50):
+                print("WWWW")
```
Comment (truher) [inline comp/vision/gamepiece_finder24.py:86]: maybe take out all these print statements

File: lib/src/main/java/org/team100/lib/config/FeedforwardConstants.java [hunk 3]
```diff
@@ -6,9 +6,9 @@ public class FeedforwardConstants {
     private final double m_kSS;
     private final double m_kDS;
 
-    //Default no friction or attached objects contraints
+    //Default no friction or attached objects contraints for NEO
     public FeedforwardConstants() {
```
Comment (truher) [inline lib/src/main/java/org/team100/lib/config/FeedforwardConstants.java:10]: don't make this a constructor, make a static method that says what it means, e.g. "public static FeedforwardConstants makeNeo() {return new FeedfowardConstants(0.122,0,0.1,0.065);}"

**Review by truher — CHANGES_REQUESTED** [review]
looks good, i made a few suggestions.  one other suggestion is to get rid of the cropping on line 133 of gamepiece_finder24.py; you want the maximum vertical size, i think.

File: lib/src/main/java/org/team100/lib/config/FeedforwardConstants.java (hunk 3 above)
Comment (Coolearthsky) [reply lib/src/main/java/org/team100/lib/config/FeedforwardConstants.java:24]: I don't know why this still says it's not outdated but I fixed this

**Pull request merged**